tests/expressions/tests.py

```diff
@@ -670,6 +670,18 @@ def test_in_lookup_allows_F_expressions_and_expressions_for_datetimes(self):
         queryset = Result.objects.filter(result_time__range=within_experiment_time)
         self.assertQuerysetEqual(queryset, ["<Result: Result at 2016-02-04 15:00:00>"])
 
+    def test_explicit_output_field(self):
+        char_field = models.CharField()
+
+        class FuncA(Func):
+            output_field = char_field
```

Comment:
`output_field = models.CharField()`
and `self.assertEqual(expr.output_field, FuncA.output_field)`?